cirq-google/cirq_google/workflow/quantum_executable_test.py

```diff
@@ -187,3 +187,12 @@ def test_quantum_executable_group_serialization(tmpdir):
     cirq.to_json(eg, f'{tmpdir}/eg.json')
     eg_reconstructed = cirq.read_json(f'{tmpdir}/eg.json')
     assert eg == eg_reconstructed
+
+
+def test_equality():
+    d1 = dict.fromkeys(['a', 'b'])
+    d2 = dict.fromkeys(['b', 'a'])
```

Comment:
why don't these dictionaries have values?

Reply:
The example from Tanuj was like this:
https://github.com/quantumlib/Cirq/issues/4734#issuecomment-995168589

But I changed the dictionaries to have keys. I also verified that the unit test still fails if I don't override the __eq__ operator.